### ISSUE TITLE
Issue: When movies are filtered, sorting by download reverts to all movies showing

### ISSUE THREAD

**Issue by tusharbhutt** [Radarr/Radarr#1553]
Whenever I filter movies to show just the monitored ones, and then sort by downloaded quality, Radarr reverts to showing all movies.  The filter for showing monitored, missing, or announced, et cetera no longer functions in Firefox until I quit.  The logs do not show any activity regarding this, and this action is repeatable in both Firefox and MS Edge.

Radarr Ver. 0.2.0.712 nightly build

**Comment by tusharbhutt** [comment]
Fair enough. However, the wanted tab only shows missing movies, not movies that exist but need to be upgraded. At least there was an issue already open for it :)

**Comment by galli-leo** [comment]
@tusharbhutt It does show exactly that under Wanted -> Cutoff Unmet :)

**Comment by tusharbhutt** [comment]
Thanks for the info, that is close to what I was looking for :) 

(I would still like to sort by downloaded resolution, but I understand your paging limitations)
